core/core-funcs.el

```diff
@@ -253,5 +253,11 @@ result, incrementing passed-tests and total-tests."
      (concat "Hidden Mode Line Mode enabled.  "
              "Use M-x hidden-mode-line-mode to make the mode-line appear."))))
 
+(defun spacemacs/recompile-elpa ()
+  "Recompile packages in elpa directory. Useful if you switch
+Emacs versions."
+  (interactive)
+  (byte-recompile-directory package-user-dir nil t))
```

Comment:
@syl20bnr If you want to make `'force` or something like that a convention I can follow it. My opinion is with eldoc and `SPC hdf` there's minimal effort in looking up an argument list, and that you should be looking it up instead of assuming from the function call what is going on (what if the function actually wants a symbol for example?).